Dockerfile

```diff
@@ -145,21 +96,30 @@ RUN if ls /tmp/resources/plugins/*.zip > /dev/null 2>&1; then \
         && mv /tmp/gs_plugin/*.jar $CATALINA_HOME/webapps/geoserver/WEB-INF/lib/ \
         && rm -rf /tmp/gs_plugin; \
       done; \
-    fi; \
-    if ls /tmp/resources/plugins/*gdal*.tar.gz > /dev/null 2>&1; then \
-    mkdir /usr/local/gdal_data && mkdir /usr/local/gdal_native_libs; \
-    unzip /tmp/resources/plugins/gdal/gdal-data.zip -d /usr/local/gdal_data && \
-    tar xzf /tmp/resources/plugins/gdal192-Ubuntu12-gcc4.6.3-x86_64.tar.gz -C /usr/local/gdal_native_libs; \
     fi;
 
+COPY ./sqljdbc4-4.0.jar $CATALINA_HOME/webapps/geoserver/WEB-INF/lib/
```

Comment:
Have Incorporated this into the PR I have done but what does this do to the image ?.